wasm/__init__.py

```diff
@@ -4586,6 +4601,8 @@ def type_func(store, funcaddr):
 def invoke_func(store, funcaddr, valstar):
     try:
         ret = spec_invoke(store, funcaddr, valstar)
+    except Trap:
+        raise
     except Exception as e:
         return store, e.args[0]
```

Comment:
Instead of updating all of the strings I've added a check to re-raise the exception here if it's exception string is one of the ones that has already been converted to use it's own exception.  This is a temporary solution that will be removed completely once all exception raising has been converted away from bare `Exception` classes.